LoopKit/Persistence/PersistenceController.swift

```diff
@@ -161,8 +159,7 @@ public final class PersistenceController {
                     options: [
                         NSMigratePersistentStoresAutomaticallyOption: true,
                         NSInferMappingModelAutomaticallyOption: true,
-                        // Data should be available on reboot before first unlock
-                        NSPersistentStoreFileProtectionKey: FileProtectionType.none
+                        NSPersistentStoreFileProtectionKey: FileProtectionType.completeUntilFirstUserAuthentication
```

Comment:
This also changes the default data protection of the .sqlite file.  After perusing the documentation, I'm not sure what CoreData does if you change the `NSPersistentStoreFileProtectionKey`.  One hopes it does something like the extension above does.

Reply:
Yeah, I'm not sure. If we really wanted to migrate the protection levels of existing files, I'm guessing we'd have to do more than just this.

Reply:
@ps2 Maybe.  This SO post (admittedly it's weak evidenced) seems to indicate that CoreData handles this:

https://stackoverflow.com/questions/36450555/changing-nspersistentstorefileprotectionkey-for-persistent-store-manager-on-exis

Would you like me to double-check before merging this?  It'd be easiest to just forge a new app and test it.

Reply:
@ps2 I have written a sample app to test the theory about whether CoreData automatically updates the file protection mode, and I have convinced myself it does.  If you want details, I'd be happy to share them.